### ISSUE TITLE
Compliance checker using old standard name table

### ISSUE THREAD

**Issue by dnowacki-usgs** [ioos/compliance-checker-web#99]
When checking a file for CF compliance at [compliance.ioos.us](https://compliance.ioos.us/), an old version of the CF Standard Name Table is used. For example:

1. Navigate to https://compliance.ioos.us/
2. Select CF-1.6
2. Enter the following OPeNDAP URL: `http://geoport.usgs.esipfed.org/thredds/dodsC/silt/usgs/Projects/stellwagen/CF-1.6/BUZZ_BAY/2651-A.cdf`
3. Note that the standard name error references v49 of the Standard Name Table, from February 2018.
<img width="824" alt="Screen Shot 2020-01-30 at 13 36 10" src="https://user-images.githubusercontent.com/13837821/73492414-80ce8f00-4365-11ea-9b88-8f0afa7a1220.png">

**Comment by Bobfrat** [comment]
Agreed, keeping up to date with the CF standard name table within the compliance checker CLI tool has been a challenge (related discussion [here](https://github.com/ioos/compliance-checker/issues/247))

One neat feature is that the compliance checker will attempt to fetch a specific version of the CF Standard Name Table if one is specified in the metadata, i.e. ```standard_name_vocabulary = "CF Standard Name Table v70```

But as you mentioned, this web frontend could/should periodically check for new versions to ensure it uses the latest version by default.

**Comment by ocefpaf** [comment]
This was improved in compliance-checker by shipping a newer table, adding a test if a new table is available, and adding a CLI option for the user to download any table they want to use. Here, in the web version, the latest one should be used and was fixed in the latest deployment. Closing this for now.
